ADR/ADR030-use-of-aws-lambda.md

```diff
@@ -0,0 +1,73 @@
+# ADR030: Use AWS Lambda small tasks run infrequently
+
+Date: 2024-05-17
+
+## Status
+
+> Accepted or Rejected
+>
+> Valid ADR statuses are: "Proposed", "Accepted", "Rejected", "Deprecated", "Superseded by ADRzzz", but since we use the GitHub Pull request mechanism to approve ADRs, "Proposed" isn't applicable.
+
+## Context
+
+There are, in this context, two types of code we want to consider
+
+1. Application code
+2. Glue code
+
+Application code is that which is written by the developers on the team, and makes up the GOV.UK Forms service and all of
+its functionality. It is executed to handle HTTP requests received by a long-lived web server process, and it has access
+to stateful data storage like PostgreSQL and Redis.
+
+Glue code is the code isn't part of any application, but is required to support  the function of the service or its 
```

Comment:
Similarly try _In contrast, glue code isn't part of any application_. Leaning into the active voice.

Reply:
Done :+1: